__tests__/app.test.js

```diff
@@ -77,6 +77,64 @@ describe("/api", () => {
             });
           });
       });
+      test("Sort order should default to date", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            const dates = reviews.map((review) => new Date(review.created_at)),
+              sortedDates = [...dates].sort((a, b) => b - a);
+            expect(dates).toEqual(sortedDates);
+          });
+      });
+      test("Should accept any valid column to sort by", () => {
```

Comment:
This would be better covered by checking that we get an error back if we're given an invalid sort_by query.